pkg/txhandler/simple/simple_transaction_handler.go

```diff
@@ -0,0 +1,473 @@
+// Copyright © 2023 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package simple
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"html/template"
+	"sync"
+	"time"
+
+	"github.com/Masterminds/sprig/v3"
+	"github.com/go-resty/resty/v2"
+	"github.com/hyperledger/firefly-common/pkg/config"
+	"github.com/hyperledger/firefly-common/pkg/ffresty"
+	"github.com/hyperledger/firefly-common/pkg/fftypes"
+	"github.com/hyperledger/firefly-common/pkg/i18n"
+	"github.com/hyperledger/firefly-common/pkg/log"
+	"github.com/hyperledger/firefly-common/pkg/retry"
+	"github.com/hyperledger/firefly-transaction-manager/internal/tmconfig"
+	"github.com/hyperledger/firefly-transaction-manager/internal/tmmsgs"
+	"github.com/hyperledger/firefly-transaction-manager/pkg/apitypes"
+	"github.com/hyperledger/firefly-transaction-manager/pkg/ffcapi"
+	"github.com/hyperledger/firefly-transaction-manager/pkg/toolkit"
+	"github.com/hyperledger/firefly-transaction-manager/pkg/txhandler"
+)
+
+// UpdateType informs FFTM whether the transaction needs an update to be persisted after this execution of the policy engine
+type UpdateType int
+
+const (
+	UpdateNo     UpdateType = iota // Instructs that no update is necessary
+	UpdateYes                      // Instructs that the transaction should be updated in persistence
+	UpdateDelete                   // Instructs that the transaction should be removed completely from persistence - generally only returned when TX status is TxStatusDeleteRequested
+)
+
+type TransactionHandlerFactory struct{}
+
+func (f *TransactionHandlerFactory) Name() string {
+	return "simple"
+}
+
+// simpleTransactionHandler is a base transaction handler forming an example for extension:
+// - It uses a public gas estimation
+// - It submits the transaction once
```

Comment:
I checked and we do now re-submit stale TX in the simple TX Handler - think this comment was stale when you moved it.

Reply:
Good catch